app/serializers/v2/profile_serializer.rb

```diff
@@ -10,6 +10,7 @@ class V2::ProfileSerializer
   belongs_to :person, serializer: ::V2::PersonSerializer
   has_many :documents, serializer: DocumentSerializer
   has_one :person_escort_record, serializer: PersonEscortRecordSerializer
+  has_many :person_escort_record_flags, serializer: FrameworkFlagSerializer
```

Comment:
Could you make this relationship lazy loaded please? I think that would mean it only gets populated when explicitly included with the new `profile.person_escort_record_flags` include on the request so the existing endpoint performance won't decrease when this doesn't get used.

Also it might be worth explicitly specifying the included fields for the `FrameworkFlagSerializer` - perhaps we can get away with not including the `belongs_to :question` relationship which would further reduce payload size and improve performance.

Reply:
thanks for the feedback; I've changed `has_one :person_escort_record` and `has_many :person_escort_record_flags` to lazy-loaded relationships.